CHANGELOG.md

```diff
@@ -27,6 +27,14 @@ Ansible Changes By Release
   the old overwriting behaviour via the config option.  In 2.5, multiple --tags
   options will be merged with no way to go back to the old behaviour.
 
+###Modules Notes:
```

Comment:
I like the idea of "Module Notes:` though I wonder if it should go *after* the list of new modules